crates/router/src/core/payments.rs

```diff
@@ -3567,17 +3652,29 @@ where
     if let Some(routing_algorithm) = request_straight_through {
         let (mut connectors, check_eligibility) = routing::perform_straight_through_routing(
             &routing_algorithm,
-            payment_data.creds_identifier.clone(),
+            payment_data.get_creds_identifier(),
         )
         .change_context(errors::ApiErrorResponse::InternalServerError)
         .attach_printable("Failed execution of straight through routing")?;
 
         if check_eligibility {
+            // let setup_mandate = payment_data.get_setup_mandate();
```

Comment:
Can we please remove these if not required?

Reply:
Sure will remove these, the changes are addressed in this https://github.com/juspay/hyperswitch/pull/5851